test/end2end_test.go

```diff
@@ -7443,13 +7444,13 @@ func doHTTPHeaderTest(t *testing.T, errCode codes.Code, headerFields ...[]string
 		headerFields: headerFields,
 	}
 	server.start(t, lis)
-	ctx, cancel := context.WithTimeout(context.Background(), 2*time.Second)
-	defer cancel()
-	cc, err := grpc.DialContext(ctx, lis.Addr().String(), grpc.WithInsecure())
+	cc, err := grpc.Dial(lis.Addr().String(), grpc.WithInsecure())
```

Comment:
Don't we need a timeout for the Dial() anymore?

Reply:
Dial() doesn't block. It returns immediately. The connection happens in the background.

Reply:
My bad.